CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Possible log types:
 
 ### UNRELEASED
 
+- [added] Completed implementation of the `Contact` field (#6)
```

Comment:
To me the [added] is just a tag, not something you should read out loud in your head :)